### PR TITLE
docs(Breadcrumbs): update README.MD

### PR DESCRIPTION
**Pull request by vrozaev**

### CODE REVIEW

File: src/components/Breadcrumbs/README.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ Breadcrumbs component. Can collapse breadcrumbs that cause overflow.
...
 | className                | `String`   |          |         | CSS class name of root element                                                                                               |
-| renderRootContent        | `Function` |          | `500`   | Custom render function of first item `(item: BreadcrumbsItem, isCurrent: boolean) => React.ReactNode;`)                      |
+| renderRootContent        | `Function` |          |         | Custom render function of first item `(item: BreadcrumbsItem, isCurrent: boolean) => React.ReactNode;`)                      |
 | renderItemContent        | `Function` |          |         | Custom render function of N+1 item `(item: BreadcrumbsItem, isCurrent: boolean, isPrevCurrent: boolean) => React.ReactNode;` |
 | renderItemDivider        | `Function` |          |         | Custom render function of items separator `() => React.ReactNode;`                                                           |
-| lastDisplayedItemsCount  | `Enum`     |    ✓     | `1`     | Number of items to display after items collapse control: `LastDisplayedItemsCount`                                           |
```
Comment (ogonkov) [inline src/components/Breadcrumbs/README.md:14]: Example below is missing this props

Comment (vrozaev) [reply src/components/Breadcrumbs/README.md:14]: fixed

File: src/components/Breadcrumbs/README.md [hunk 2]
```diff
@@ -33,5 +33,5 @@ const breadcrumbs = [
   },
 ];
 
-return <Breadcrumbs items={items} />;
+return <Breadcrumbs items={items} lastDisplayedItemsCount={LastDisplayedItemsCount.One} firstDisplayedItemsCount={FirstDisplayedItemsCount.One} />;
```
Comment (ValeraS) [inline src/components/Breadcrumbs/README.md:36]:
```suggestion
return (
    <Breadcrumbs
        items={items}
        lastDisplayedItemsCount={LastDisplayedItemsCount.One}
        firstDisplayedItemsCount={FirstDisplayedItemsCount.One}
    />
);
```

File: src/components/Breadcrumbs/README.md (hunk 1 above)
Comment (ogonkov) [reply src/components/Breadcrumbs/README.md:14]: Lint failed

**Comment by vrozaev** [comment]
@ValeraS all issues fixed

**Pull request merged**